plugins/base/src/main/kotlin/translators/psi/DefaultPsiToDocumentableTranslator.kt

```diff
@@ -427,7 +455,8 @@ class DefaultPsiToDocumentableTranslator(
                 },
                 documentation = docs.toSourceSetDependent(),
                 expectPresentInSet = null,
-                sources = PsiDocumentableSource(psi).toSourceSetDependent(),
+                // isPhysical detects the virtual methods without real sources. Here it used for default constructor
+                sources = if (psi.isPhysical) PsiDocumentableSource(psi).toSourceSetDependent() else emptyMap(),
```

Comment:
Nice! If you have the time, let's fix it for other documentables as well? There's 3 usages total

Could do the same type of extension as proposed above, something along the lines of:

```kotlin
fun PsiNamedElement.parseSources(): SourceSetDependent<DocumentableSource> {
    return when {
        psi.isPhysical -> PsiDocumentableSource(this).toSourceSetDependent()
        else -> emptyMap()
    }
}
```